### ISSUE TITLE
Tag CSS Color on Posts

### ISSUE THREAD

**Issue by justinrummel** [mmistakes/hpstr-jekyll-theme#77]
I'm not a CSS expert, but was confused when I found in the _page.scss for the .tag section, there are two lines for "color:"

https://github.com/mmistakes/hpstr-jekyll-theme/blob/master/_sass/_page.scss#L175-177

Also, the default color is $white which does not display with a white background. If you replace one of the $white with $text-color it does display, however, I want to make sure these are the right lines of CSS to modify, thus: 

``` bash
//		color: $white;
		@include rounded(3px);
		color: $text-color;
```

**Comment by mmistakes** [comment]
There should only be one `color: $white;` declaration. You can safely remove one.

![tag-pills](https://cloud.githubusercontent.com/assets/1376749/5729423/48955628-9b40-11e4-9ed0-4d240b25e801.png)

The current style is white text on a gray background for the tag "pills". Here are the important parts you'll want to modify to change the colors.

``` css
.tag {
	color: $white; // tag text color
	.term {
		background-color: lighten($base-color,60); // background color of tag name text
	}
	.count {
		background-color: lighten($base-color,40); // background color of tag count to the right of the term name
	}
	&:hover {
		background-color: lighten($base-color,50); // background color when tag link is hovered
	}
}
```

**Comment by justinrummel** [comment]
You are correct that is the display in the /tags/ pages, however, you have enabled tags at the bottom of posts per https://github.com/mmistakes/hpstr-jekyll-theme/blob/master/_layouts/post.html#HL46 

These tags do not display as it's a white link on a white background.  I can't just use the grey "pill" in the background (via ```<span class="term">```) as it's rounded on the left and not the right. I've added the dark "count" section (via ```<span class="count">&nbsp;</span>```), but still not sure if I like the end results.

``` html
<span class="entry-tags">{% for tag in page.tags %}<a href="{{ site.url }}/tags/#{{ tag }}" title="Pages tagged {{ tag }}" class="tag"><span class="term">{{ tag }}</span><span class="count">&nbsp;</span></a>{% unless forloop.last %}{% endunless %}{% endfor %}</span>
```

**Comment by justinrummel** [comment]
Oh, for the example above I did return ```color: $white``` back to the original value.

**Comment by mmistakes** [comment]
Got it. When I converted the stylesheets from Less to Sass a while back I missed some styles. Tag links on posts should show up just fine now. Check out my [lastest commit](https://github.com/mmistakes/hpstr-jekyll-theme/commit/1fdd5f0b7025f5eabb815737886ef79baf83df8f) to see what I changed.

![post-tags](https://cloud.githubusercontent.com/assets/1376749/5739712/7602c9bc-9bca-11e4-9e9d-a8b5a390cd3a.png)
